ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -126,12 +126,14 @@ func NewReconciler(ctx context.Context, mgr manager.Manager, options RayClusterR
 		rayClusterScaleExpectation: expectations.NewRayClusterScaleExpectation(mgr.GetClient()),
 		headSidecarContainers:      options.HeadSidecarContainers,
 		workerSidecarContainers:    options.WorkerSidecarContainers,
+		PreStopCommandList:         options.PreStopCommandList,
```

Comment:
It might be better to pass the entire set of options to the reconciler instead of passing them one by one, but that’s not related to this PR. It’s not necessary to update it here.